src/Classes/ServiceAPI/MyRadio_User.php

```diff
@@ -2402,7 +2469,9 @@ public function toDataSource($mixins = [])
         $data = [
             'memberid' => $this->getID(),
             'fname' => $this->getFName(),
+            'nname' => $this->getNName(),
             'sname' => $this->getSName(),
+            //Warning this will leak user emails to public as the api isn't secure
```

Comment:
this comment feels badly worded?

Reply:
this was a hold over me investigating what causes public emails to be displayed the comment likely can be removed